test/selector.jl

```diff
@@ -1016,24 +1016,6 @@ end
         end
     end
 
-    @testset "single-arity views" begin
```

Comment:
Are we losing code coverage from deleting these?

Reply:
It doesn't seem so (diff coverage is 100% and total coverage is increased), but I've reverted the removal and instead fixed the tests.